component/src/main/java/org/wso2/extension/siddhi/store/elasticsearch/ElasticsearchEventTable.java

```diff
@@ -250,7 +259,20 @@
                 @Parameter(name = "trust.store.pass",
                         description = "Trust store password.",
                         type = {DataType.STRING}, optional = true,
-                        defaultValue = "wso2carbon")
+                        defaultValue = "wso2carbon"),
+                @Parameter(name = "index.request.timeout",
+                        description = "Timeout to wait for the all the nodes to " +
+                                "acknowledge the index-related operations in seconds.",
+                        type = {DataType.LONG}, optional = true,
+                        defaultValue = "null"),
```

Comment:
defaultValue of `null` for a `DataType.LONG` seems a bit odd. Can we have a long (eg: `0` or `1`)?
For example, check the `bulk.size` parameter above.

Reply:
Yes we have set of default values for timeouts. I forgot to add those here.

Fixed with https://github.com/siddhi-io/siddhi-store-elasticsearch/pull/63/commits/348a8cb9a1acf5c9eda57314f51665e04dd247b4